opentok/opentok-meet
# Fix empty room name bug

#### What is this PR doing?
If you click 'join' while the 'room name' field is empty and then attempt to join a properly named room you will be unable to due to the `tokenRole` query blocking the room name in the url. It's super minor bug but it's been annoying me.

#### How should this be manually tested?
On master:
1. Navigate to meet landing page. Click `join` while the `room name` field is empty. 
2. Observe that the url is now the domain + `?tokenRole=moderator`
3. Enter something into the room name field and try to join again. Notice that you're now stuck on the login page.

On this PR:
1. Navigate to meet landing page. Click `join` while the `room name` field is empty. 
2. Observe that the url is now the domain + `?tokenRole=moderator`
3. Enter something into the room name field and try to join again. Notice that you're now properly in a room

#### What are the relevant tickets?
